runtime/queries/markdown_inline/highlights.scm

```diff
@@ -0,0 +1,91 @@
+;; From MDeiml/tree-sitter-markdown
+[
+  (code_span)
+  (link_title)
+] @text.literal
+
+[
+  (emphasis_delimiter)
+  (code_span_delimiter)
+] @punctuation.delimiter
+
+(emphasis) @text.emphasis
```

Comment:
Nvim-treesitter uses `@text` for everything that is not programming-related, while Helix uses `@markup` for such... well, markup -- which I kinda prefer. We need to decide whether we want to make up our own minds on this, or follow nvim-treesitter blindly.